project/Dependencies.scala

```diff
@@ -130,13 +136,13 @@ object Dependencies {
   )
 
   val playTestkit = l ++= Seq(
-    Compile.play
-  ) ++ (Seq(
     Compile.play,
-    Test.playTest,
-    Test.playSpecs2,
-    Test.scalaTestPlusPlay
-  ).map(_.withConfigurations(Some("provided"))))
```

Comment:
I'm not sure what the thinking was here, so if anyone remembers, let me know. 🙂

Reply:
I suspect the thinking was to avoid pulling in specs2 for scalatest users, avoid pulling in scalatest for specs2 users, etc. So it looks like that's indeed no longer necessary in the new structure :+1:

Reply:
That sounds plausible! 👍